devito/symbolics/printer.py

```diff
@@ -318,14 +356,66 @@ def _print_Fallback(self, expr):
 
 # Lifted from SymPy so that we go through our own `_print_math_func`
 for k in ('exp log sin cos tan ceiling floor').split():
-    setattr(CodePrinter, '_print_%s' % k, CodePrinter._print_math_func)
+    setattr(_DevitoPrinterBase, '_print_%s' % k, _DevitoPrinterBase._print_math_func)
 
 
 # Always parenthesize IntDiv and InlineIf within expressions
 PRECEDENCE_VALUES['IntDiv'] = 1
 PRECEDENCE_VALUES['InlineIf'] = 1
 
 
+# Sympy 1.11 has introduced a bug in `_print_Add`, so we enforce here
+# to always use the correct one from our printer
+if Version(sympy.__version__) >= Version("1.11"):
+    setattr(sympy.printing.str.StrPrinter, '_print_Add', _DevitoPrinterBase._print_Add)
+
+
+class CDevitoPrinter(_DevitoPrinterBase, C99CodePrinter):
```

Comment:
I don't think we need "Devito" in all these class names

Reply:
You could always abbreviate to `Dv` if really necessary